mkosi.conf.d/opensuse/mkosi.conf

```diff
@@ -0,0 +1,68 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=opensuse
+
+[Distribution]
+Release=tumbleweed
+
+[Config]
+Dependencies=initrd
+
+[Content]
+Initrds=%O/initrd
+Packages=
+        bpftool
+        # SSL CA needed for zypper
+        ca-certificates
+        ca-certificates-mozilla
+        cryptsetup
+        distribution-gpg-keys
+        git-core
+        iproute2
+        iputils
+        kernel-default
+        kernel-firmware-all
```

Comment:
We don't pull in firmware for the default profile in any of the other distros, so don't do this here either. Move this to the desktop profile like we do for the other distros.